.tito/lib/origin/builder/__init__.py

```diff
@@ -55,15 +55,37 @@ def _setup_test_specfile(self):
             )
             # Custom Openshift v3 stuff follows, everything above is the standard
             # builder
-            cmd = '. ./hack/common.sh ; echo $(os::build::ldflags)'
+            
+            ## Fixup os_git_vars
```

Comment:
I now understand what you are saying, but I'd like to keep the logic where it is for this pull request.

I have some ideas for building srpm's, #7268 and I'd like to defer this cleanup until then.